modules/nf-core/svtools/vcftobedpe/main.nf

```diff
@@ -0,0 +1,44 @@
+process SVTOOLS_VCFTOBEDPE {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/svtools:0.5.1--py_0':
+        'biocontainers/svtools:0.5.1--py_0' }"
+
+    input:
+    tuple val(meta), path(vcf)
+
+    output:
+    tuple val(meta), path("*.bedpe"), emit: bedpe
+    path "versions.yml"             , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    svtools vcftobedpe \\
```

Comment:
Missing $args (even if there are no more, who knows what the might add in the future)
```suggestion
    svtools vcftobedpe \\
        $args
```

Reply:
There are no additional args for this module :)